packages/flutter/lib/src/rendering/mouse_tracking.dart

```diff
@@ -332,10 +332,16 @@ abstract class BaseMouseTracker extends ChangeNotifier {
   LinkedHashMap<MouseTrackerAnnotation, Matrix4> _hitTestResultToAnnotations(HitTestResult result) {
     assert(result != null);
     final LinkedHashMap<MouseTrackerAnnotation, Matrix4> annotations = <MouseTrackerAnnotation, Matrix4>{}
-        as LinkedHashMap<MouseTrackerAnnotation, Matrix4>;
+      as LinkedHashMap<MouseTrackerAnnotation, Matrix4>;
     for (final HitTestEntry entry in result.path) {
-      if (entry.target is MouseTrackerAnnotation) {
-        annotations[entry.target as MouseTrackerAnnotation] = entry.transform!;
+      final HitTestTarget target = entry.target;
+      if (target is MouseTrackerAnnotation) {
+        if (target is RenderObject) {
+          // We should ignore the detached renderObject, otherwise, may trigger using the disposed object.
+          if (!target.attached)
```

Comment:
!attached is not quite the same as disposed.

@goderbauer WDYT of this?

Reply:
Yes, what you pointed out is correct, the range of `detached` is greater than and includes `disposed`, so here we need to consider whether it is reasonable to ignore the detached renderObject mouse event handling

Reply:
I think this behavior is reasonable. If an RO has detached from the tree, it should not respond to any mouse events. The particularity of this problem is that the RO was detached after the `hittest`, so it is reasonable and controllable to ignore the handling of this event here. Additionally, if the RO is retaken later for some reasons, this change will not affect it, because it is a real-time conditional control and does not remove the callback.

Reply:
That makes sense to me - we may want to update the comment to explain this rather than refering to disposal, which isn't really a thing for RenderObjects.

I'd also defer to @goderbauer or @dkwingsmt on this - I think they're more familiar with this area.